MiniSpace.Services.Notifications/src/MiniSpace.Services.Notifications.Application/Dto/StudentDto.cs

```diff
@@ -9,16 +9,20 @@ public class StudentDto
         public string Email { get; set; }
         public string FirstName { get; set; }
         public string LastName { get; set; }
-        public int NumberOfFriends { get; set; }
-        public Guid ProfileImage { get; set; }
+        public string ProfileImageUrl { get; set; }
         public string Description { get; set; }
-        public DateTime DateOfBirth { get; set; }
+        public DateTime? DateOfBirth { get; set; }
         public bool EmailNotifications { get; set; }
         public bool IsBanned { get; set; }
-        public bool IsOrganizer { get; set; }
         public string State { get; set; }
         public DateTime CreatedAt { get; set; }
-        public List<Guid> InterestedInEvents { get; set; }
-        public List<Guid> SignedUpEvents { get; set; }
+        public string ContactEmail { get; set; }
+        public string BannerUrl { get; set; }
```

Comment:
Didn't you have the separated repositories for events interested and signed-up?

Reply:
Maybe for the purpose of future requirements it will be updated. Currently  this is not optimized.